resources/views/shared/footer.blade.php

```diff
@@ -268,6 +268,12 @@ function closedCallback(orderObjOrNull) {
         var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0]; g.type='text/javascript';
         g.defer=true; g.async=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
     })();
+
+    setTimeout(function() {
+      if ($("html[class*='gr__']").length) {
+        _paq.push(['trackEvent', 'grammarly-user', '1']);
```

Comment:
What does this do exactly? 
The PR had no description.

Reply:
It checks if the user has grammarly installed and if so logs that to your analytics tool (Matomo/Piwik).